src/RealizationOrder.h

```diff
@@ -20,9 +21,14 @@ class Function;
  * order in which to do the scheduling. This in turn influences the
  * order in which stages are computed when there's no strict
  * dependency between them. Currently just some arbitrary depth-first
- * traversal of the call graph. */
-std::vector<std::string> realization_order(const std::vector<Function> &output,
-                                           const std::map<std::string, Function> &env);
+ * traversal of the call graph. In addition, determine grouping of functions
+ * which loop are to be fused together. The functions within the fused groups
+ * are sorted based on realization order. There should not be any dependencies
+ * among functions within a fused group. This pass will also populate the
+ * 'fused_pairs' list in the function's schedule.
+ */
```

Comment:
Would be good to explicitly explain the first and second members of the returned pair.